rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ListJobCommand.java

```diff
@@ -64,52 +61,22 @@ public ListJobCommand(String... filtering) {
     @Override
     public void execute(ApplicationContext currentContext) throws CLIException {
         try {
-            if (isIteractiveMode(currentContext)) {
-                int numOfJobsPerPage = getNumOfJobsPerPage(currentContext);
-                int numOfJobs = getNumberOfJobs(beginIndex, offset, currentContext);
-                if (numOfJobs < numOfJobsPerPage) {
-                    printJobsList(beginIndex, numOfJobs, currentContext);
-                } else {
-                    int pages = (numOfJobs / numOfJobsPerPage) + ((numOfJobs % numOfJobsPerPage) == 0 ? 0 : 1);
-                    int iterCount = 0;
-                    boolean hasQuit = false;
-                    // all pages except the last one
-                    for (; iterCount < pages - 1; iterCount++) {
-                        printJobsList(beginIndex + (iterCount * numOfJobsPerPage), numOfJobsPerPage, currentContext);
-                        String response = readLine(currentContext,
-                                                   "Page(%d/%d). Press ENTER to continue. 'q' to quit. ",
-                                                   (iterCount + 1),
-                                                   pages);
-                        if ("q".equalsIgnoreCase(response)) {
-                            hasQuit = true;
-                            break;
-                        }
-                    }
-                    if (!hasQuit) {
-                        // last page
-                        printJobsList(beginIndex + (iterCount * numOfJobsPerPage),
-                                      numOfJobs - (iterCount * numOfJobsPerPage),
-                                      currentContext);
-                    }
-                }
-            } else {
-                printJobsList(beginIndex, (offsetFlag) ? offset : RestConstants.DFLT_PAGINATION_SIZE, currentContext);
-            }
```

Comment:
Great ! I was thinking about the following use-case actually:
1) I run ./proactive-client
2) listjobs(200)
3) I read the first page
4) I press enter to see the next page
5) I read the second page
6) etc...

I believe step 4) and beyond are not possible anymore. But I don't think it's a major problem since the other listing commands are also not behaving like that anyway.